examples/grpc_example/MODULE.bazel

```diff
@@ -0,0 +1,66 @@
+bazel_dep(
+    name = "rules_swift_package_manager",
+    version = "0.0.0",
+)
+local_path_override(
+    module_name = "rules_swift_package_manager",
+    path = "../..",
+)
+
+bazel_dep(name = "cgrindel_bazel_starlib", version = "0.16.2")
+bazel_dep(name = "bazel_skylib", version = "1.4.2")
+bazel_dep(
+    name = "rules_swift",
+    version = "1.9.1",
+    repo_name = "build_bazel_rules_swift",
+)
+
+non_module_deps = use_extension("@build_bazel_rules_swift//swift:extensions.bzl", "non_module_deps")
+use_repo(
+    non_module_deps,
+    "build_bazel_rules_swift_index_import",
+    "build_bazel_rules_swift_local_config",
+    "com_github_apple_swift_log",
+    "com_github_apple_swift_nio",
+    "com_github_apple_swift_nio_extras",
+    "com_github_apple_swift_nio_http2",
+    "com_github_apple_swift_nio_transport_services",
+    "com_github_apple_swift_protobuf",
+    "com_github_grpc_grpc_swift",
+)
+
+bazel_dep(
+    name = "rules_apple",
+    version = "2.5.0",
+    repo_name = "build_bazel_rules_apple",
+)
+
+bazel_dep(
+    name = "bazel_skylib_gazelle_plugin",
+    version = "1.4.2",
+    dev_dependency = True,
+)
+bazel_dep(
+    name = "gazelle",
+    version = "0.32.0",
+    dev_dependency = True,
+    repo_name = "bazel_gazelle",
+)
+bazel_dep(name = "rules_proto", version = "5.3.0-21.7")
+
+# TODO: Gazelle currently needs protobuf to be named com_google_protobuf.
```

Comment:
Let's remove the `TODO: ` part of this comment. I think that the rest of the comment says exactly what is needed.

FYI. I try to avoid `TODO` comments being merged. That is why I prefer that we create GH issues and use `GHXXX` comments instead.